payloads/Makefile

```diff
@@ -0,0 +1,2 @@
+OREBOOT=../
```

Comment:
All of these declarations need to be of the form `$(abspath $(CURDIR)/..)`. Just using plain relative paths won't work.

Reply:
fixed